packages/core/src/layout/HomepageTimer/HomepageTimer.tsx

```diff
@@ -14,59 +14,88 @@
  * limitations under the License.
  */
 
-import React, { FC } from 'react';
+import React from 'react';
 import { HeaderLabel } from '../HeaderLabel';
+import { useApi, configApiRef } from '@backstage/core';
```

Comment:
Import these from `@backstage/core-api`. The core package is the only place you're allowed to do that :grin: